# 로봇 초기 위치
init_pose_robot:
  position: [-1.06122060474, 0.169611258379, 0.0]
  orientation: [0.0, 0.0, 0.0388178384485, 0.99924630368]
# 쟁반 테이블
table_order:
  position: [-0.137923970891, 0.155171041106, 0.0]
  orientation: [0.0, 0.0, 0.0457271296998, 0.998953967713]
# 1번 테이블
table_pose_first:
  position: [0.649043034533, 0.0546112149981, 0.0]
  orientation: [0.0, 0.0, 0.0555141260648, 0.99845790187]
# 쟁반 회수대  
table_return:
  position: [-0.829604404893, -0.0155230989438, 0.0]
  orientation: [0.0, 0.0, 0.998385922902, -0.0567939164922]
